docs/src/main/sphinx/release/release-418.md

```diff
@@ -0,0 +1,62 @@
+# Release 418 (17 May 2023)
+
+## General
+
+* Add support for [EXECUTE IMMEDIATE](/sql/execute-immediate). ({issue}`17341`)
+* Fix failure when invoking `current_timestamp`. ({issue}`17455`)
+
+## BigQuery connector
+
+* Add support for adding labels to BigQuery jobs started by Trino as part of
+  query processing. The name and value of the label can be configured via the
+  `bigquery.job.label-name` and `bigquery.job.label-format` catalog
+  configuration properties, respectively. ({issue}`16187`)
+
+## Delta Lake connector
+
+* Add support for `INSERT`, `UPDATE`, `DELETE`, and `MERGE` statements for
+  tables with an `id` column mapping. ({issue}`16600`)
+* Add the `table_changes` table function. ({issue}`16205`)
+* Improve performance of joins on partitioned columns. ({issue}`14493`)
+
+## Hive connector
+
+* Improve performance of querying `information_schema.tables` when using the
+  Hive metastore. ({issue}`17127`)
+* Improve performance of joins on partitioned columns. ({issue}`14493`)
+* Improve performance of reading Parquet files by enabling the optimized Parquet 
```

Comment:
How is this related to improving performance of *reads*? Shouldn't this be "Improve performance of *writing* parquet data by enabling ..." ?

Reply:
It can potentially improve reads due to https://github.com/trinodb/trino/pull/17404 but the main benefit should be better memory usage accounting and speed of writer